src/components/TodoList/TodoList.tsx

```diff
@@ -1,100 +1,129 @@
-import React from 'react';
+import React, { useMemo } from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
 
-export const TodoList: React.FC = () => (
-  <table className="table is-narrow is-fullwidth">
-    <thead>
-      <tr>
-        <th>#</th>
-        <th>
-          <span className="icon">
-            <i className="fas fa-check" />
-          </span>
-        </th>
-        <th>Title</th>
-        <th> </th>
-      </tr>
-    </thead>
+type Props = {
+  todos: Todo[];
+  fnSelectTodo: (todoId: number) => void;
+  selectTodoId: number;
+  fnSelectUser: (selectUserId: number) => void;
+  status: StatusOfFilter | string;
```

Comment:
Instead of using 'StatusOfFilter | string', you should only use 'StatusOfFilter' as a type for the 'status' prop.